exercises/bracket-push/bracket_push.py

```diff
@@ -1,2 +1,2 @@
-def check_brackets(input_string):
```

Comment:
I think that `input_string` is more descriptive. To me, `value` is too generic. I would prefer that we leave this as-is.